Packages/Core/CoreSession/Sources/CoreSession/Account creation/AccountCreationHandler.swift

```diff
@@ -32,15 +32,15 @@ public struct AccountCreationHandler {
         self.accountService = accountService
     }
 
-    public func accountCreationMethodAvailibility(for login: Login, context: LoginContext? = nil, completion: @escaping CompletionBlock<(AccountCreationMethodAvailibility?), Error>) {
+    public func accountCreationMethodAvailability(for login: Login, context: LoginContext? = nil, completion: @escaping CompletionBlock<(AccountCreationMethodAvailability?), Error>) {
         self.accountService.verifyExistence(of: login) { response in
             switch response {
             case let .success(accountInfo):
-                let accountCreationMethodAvailibility = AccountCreationMethodAvailibility(isLoginAvailable: !accountInfo.isAccountRegistered,
+                let accountCreationMethodAvailability = AccountCreationMethodAvailability(isLoginAvailable: !accountInfo.isAccountRegistered,
                                                                                           shouldRegisterViaSSO: accountInfo.sso,
                                                                                           serviceProviderURL: accountInfo.ssoServiceProviderUrl, isNitroProvider: accountInfo.ssoIsNitroProvider ?? false,
                                                                                           context: context)
-                completion(.success(accountCreationMethodAvailibility))
+                completion(.success(accountCreationMethodAvailability))
             case let .failure(error):
                 completion(Result.failure(error))
             }
```

Comment:
Comment for renamed file below...

```diff
-SQLIte
+SQLite
```

warning:  renames like this can be problematic on file systems that aren't case sensitive...